app/src/bin/crate.in.sh

```diff
@@ -101,7 +101,7 @@ JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8"
 JAVA_OPTS="$JAVA_OPTS -Djna.nosys=true"
 
 # log4j options
-JAVA_OPTS="$JAVA_OPTS -Dlog4j.shutdownHookEnabled=false -Dlog4j2.disable.jmx=true -Dlog4j.skipJansi=true"
+JAVA_OPTS="$JAVA_OPTS -Dlog4j.configurationFile=\"$CRATE_HOME/config/log4j2.properties\" -Dlog4j.shutdownHookEnabled=false -Dlog4j2.disable.jmx=true -Dlog4j.skipJansi=true"
```

Comment:
I think we can add the config directory to the classpath. log4j supports discovering the config from the classpath.

Reply:
Ah ok, we don't have the config directory because we set it via `-Cpath.conf`. I think we can go through the `log4j.configurationFile` property then when we initialize the config.

Reply:
@mxm that's happening in the bootstrapping already afaik

Reply:
@chaudum No, doesn't seem to be the case.